entrypoints/content/components/secondary/comment/CommentAction.tsx

```diff
@@ -0,0 +1,71 @@
+
+import { useState } from 'react'
+
+
+// Components::
+import { ArrowBigDown, ArrowBigUp, Ellipsis, Forward, MessageSquare } from 'lucide-react'
+
+// Types:
+import { VoteCount } from 'types/votes'
+
+interface VoteCountProps {
+  voteCount: VoteCount
+}
+
+const CommentAction: React.FC<VoteCountProps> = ({ voteCount: { up = 0, down = 0 } }) => {
+
+  const [voteCount, setVotecount] = useState(up - down)
+  const [isUpVoteActive, setUpVoteActive] = useState(true)
+  const [isDownVoteActive, setDownVoteActive] = useState(false)
+
+  useEffect(() => {
+    setVotecount(up - down)
+  }, [up, down])
```

Comment:
a little redundant, because ultimately `voteCount` is just a simple difference, so we can just **not** create a state variable for this and directly render/use `upvote - downvote`

Reply:
Let's not worry about this rn. We can fix this when we integrate backend